magda-web-server/src/getIndexFileContent.ts

```diff
@@ -92,6 +94,16 @@ async function getIndexFileContent(
             "</head>",
             `<link href="/api/v0/content/stylesheet.css" rel="stylesheet">\n</head>`
         );
+
+        if (uiBaseUrl === "/") {
+            // --- if `uiBaseUrl` is '/' do nothing
+            return indexFileContent;
+        }
+
+        indexFileContent = indexFileContent.replace(
+            '<base href="/">',
+            `<base href="${uiBaseUrl}/">`
```

Comment:
Sorry I missed that the trailing slash gets appended here!

Reply:
I still think it's confusing if we append it here though - when you think about the default value of `/`, that itself can work in all cases (e.g. concatenate it with other paths) - whereas if you give it `/magda`, suddenly you have to treat it differently without doing extra stuff to standardise it anywhere you end up using it